README.md

```diff
@@ -268,3 +289,52 @@ Run the `pytest` module to run the coverage tests:
 ```bash
 $> python -m pytest --cov-report=html --cov=history tests/
 ```
+
+# **Dependencies**
+
+The service dependencies are listed in the next topics.
+
+- _Dojot Services_: They are dojot services;
+- _Others Services_: They are external services;
+
+### **Dojot Services**
+
+  - [Auth](https://github.com/dojot/auth);
+  - [Device Manager](https://github.com/dojot/device-manager);
+  - [Data Broker](https://github.com/dojot/data-broker);
+
+### **Others Services**
+
+  - Kafka (tested using Kafka version 2.12);
```

Comment:
Isn't Zookeeper a dependency too?

Reply:
Good question!

Reply:
I think not... 
I put it in the flowbroker because the flowbroker context manager uses zookeeper...

Reply:
In my opinion, the zookeeper is a dependency on kafka and not on History, but as I'm not sure of that, it also includes it in the README

Reply:
Should I remove it then? 
let's solve this in jokenpo :fist: :raised_hand: :v:

Reply:
It doesn't make sense to me that Zookeeper is here ...

Reply:
I commented only because I know they are related to each other, but I don't know if they need to be together always

Reply:
Yes, kafka always depends on the zookeeper...